src/main/kotlin/racing/CarRacingResultView.kt

```diff
@@ -1,21 +1,26 @@
 package racing
 
-class CarRacingResultView {
+object CarRacingResultView {
     fun printCarRacingResult(cars: List<Car>, moveCount: Int) {
         // turnOfPosition의 0번 index는 기본으로 1이 셋팅되어있기때문에
         // 범위를 0 <= moveIndex <= moveCount 로 설정
         for (moveIndex in 0..moveCount) {
-            cars.forEach { car ->
-                printCurrentCarMovingDistance(car.turnOfPosition[moveIndex])
-            }
-            println()
+            printMovingDistanceByTurn(cars = cars, moveIndex = moveIndex)
         }
     }
 
-    private fun printCurrentCarMovingDistance(carPosition: Int) {
-        for (i in 0 until carPosition) {
-            print("_ ")
+    private fun printMovingDistanceByTurn(cars: List<Car>, moveIndex: Int) {
+        cars.forEach { car ->
+            printCurrentCarMovingDistance(carName = car.name, carPosition = car.turnOfPosition[moveIndex])
```

Comment:
```suggestion
            print(car = car.name, distance = car.turnOfPosition[moveIndex])
```
함수 이름과 네임드 파라메터를 잘 조합하면 꽤 좋은 가독성을 얻을 수 있습니다.

Reply:
출력 함수명에 모든 것을 다 쓰지 않고, 네임드 파라미터를 이용해서 해당 출력함수가 어떤 기능을 하는지 알 수 있게 하는 것도 좋다라고 말씀하시는걸까요?!

Reply:
네 맞아요.

Reply:
아하! 이해했습니다. 감사합니다 ㅎㅎ

그렇다면 printCurrentCarMovingDistance 외에 다른 함수들도 네임드 파라미터를 이용해서 전부 print로 바꾸는 것도 가독성에 도움이 되는 일일까요?

네임드 파라미터를 이용하다보니 print 함수의 내용을 의도대로 사용할 수 있을 것 같아, print로 바꿔서 오버로딩 사용해도 괜찮을 것 같긴한데.. 파라미터 타입이 같은 함수 케이스들도 생길 수 있을 것 같아서요!

전부 바꾸지 않는다면 네임드 파라미터를 활용하여 print 함수로 변경하는 기준은 어떤 것으로 세우는 것이 좋을지 조언 부탁드릴 수 있을까요?!

Reply:
> 그렇다면 printCurrentCarMovingDistance 외에 다른 함수들도 네임드 파라미터를 이용해서 전부 print로 바꾸는 것도 가독성에 도움이 되는 일일까요?

위 제안은 함수 네이밍을 하실 때 추가로 고려할 수 있는 부분을 구체적으로 설명하기 위해 든 예시입니다.
모든 메소드 이름을 `print` 로 하고 인자에 따라 오버로딩 하면 더 이상할 것 같아요.ㅎㅎ

함수 네이밍에 대한 조언을 더 드리고 싶은데 마지막 단계 코드리뷰에서 조금 더 구체적으로 디테일하게 드려보도록 하겠습니다.